src/offset/utc.rs

```diff
@@ -98,3 +112,98 @@ impl fmt::Display for Utc {
         write!(f, "UTC")
     }
 }
+
+#[cfg(all(feature = "clock", feature = "test-override"))]
+thread_local!(static OVERRIDE: std::cell::RefCell<Option<DateTime<Utc>>> = std::cell::RefCell::new(None));
```

Comment:
Same request for a code comment.

Same suggestion for name change to `UTC_NOW_OVERRIDE`, which also will disambiguate from the other `OVERRIDE`.